programs/perpetuals/src/instructions/remove_liquidity.rs

```diff
@@ -201,11 +221,15 @@ pub fn remove_liquidity(
 
     // update pool stats
     msg!("Update pool stats");
-    pool.aum_usd = pool_amount_usd.saturating_sub(
-        token_price
-            .get_token_amount(withdrawal_amount, custody.decimals)?
-            .into(),
-    );
+    pool.aum_usd = if custody.pricing.use_ema {
+        pool.get_assets_under_management_usd(AumCalcMode::Last, ctx.remaining_accounts, curtime)?
+    } else {
+        pool_amount_usd.saturating_sub(
+            token_price
+                .get_token_amount(withdrawal_amount, custody.decimals)?
+                .into(),
+        )
```

Comment:
what's the difference between these 2 calculations?

Reply:
What is happening here - as I described above, `AumCalcMode::Last` should be used to record `pool.aum_usd`, but if pricing.use_ema == false, then all modes return the same value, so there is no need to recalculate AUM from scratch and just adjust the value with the newly deposited amount (to save on compute units). 
I agree, it is puzzling, maybe I should remove that second branch or comment the thing.

Reply:
why should AumCalcMode::Last be the mode to calculate when pricing.use_ema == true? 

should we enforce all token to `use_ema` = true after recent usdc incident